backend/compact-connect/docs/README.md

```diff
@@ -21,17 +21,14 @@ This system includes a mock license data API that has two functions: A synchrono
 ## How to use the API bulk-upload feature
 [Back to top](#compact-connect---technical-user-guide)
 
-The data system API supports uploading of a large CSV file for asynchronous data ingest. The feature involves using two endpoints, which are described in the [Open API Specification](#open-api-specification) above. To upload a file for asynchronous data ingest perform the following steps:
-1) Export your license data to a CSV file, formatted as follows:
+## Generating a CSV export of your license data
```

Comment:
Nit: can we make this an h3 since it's a subsection of the above? And same thing for the below sections on Manual Uploads and Machine-to-machine